spec/mainadditions.html

```diff
@@ -514,9 +514,9 @@ <h1>
         1. Let _parseResult_ be ParseText(StringToCodePoints(_offsetString_), <del>|UTCOffset|</del><ins>|UTCOffsetSubMinutePrecision|</ins>).
         1. <del>Assert: _parseResult_ is not a List of errors.</del>
         1. <ins>If _parseResult_ is a List of errors, throw a *RangeError* exception.</ins>
-        1. Assert: _parseResult_ contains a |TemporalSign| Parse Node.
-        1. Let _parsedSign_ be the source text matched by the |TemporalSign| Parse Node contained within _parseResult_.
-        1. If _parsedSign_ is the single code point U+002D (HYPHEN-MINUS) or U+2212 (MINUS SIGN), then
+        1. Assert: _parseResult_ contains a |ASCIISign| Parse Node.
+        1. Let _parsedSign_ be the source text matched by the |ASCIISign| Parse Node contained within _parseResult_.
+        1. If _parsedSign_ is the single code point U+002D (HYPHEN-MINUS), then
```

Comment:
These would be included in the ECMA-262 PR, yes?

Reply:
If you're asking whether https://github.com/tc39/ecma262/pull/3334 also includes replacing "TemporalSign" with "ASCIISign", then yes it does.